integration-tests/src/test/scala/tech/beshu/ror/integration/proxy/FieldLevelSecurityProxyTests.scala

```diff
@@ -16,7 +16,6 @@
  */
 package tech.beshu.ror.integration.proxy
 
-import tech.beshu.ror.integration.suites.FieldLevelSecuritySuite
 
 // todo: enable when proxy supports fields rule
```

Comment:
ATM proxy won't support Lucene level fields filtering so it'd be nice to have several modes for ROR FLS:
* old mode (FLS at Lucene level)
* new mode (QUERY modifications + fallback to old mode)
* proxy mode (QUERY modification + forbid)

Are we able to do it in a pretty simple way, or we should create a separate JIRA for proxy?

Reply:
let's create a jira.